extensions/oidc-client-reactive-filter/runtime/src/main/java/io/quarkus/oidc/client/reactive/filter/runtime/OidcClientReactiveFilterConfigFallbackInterceptor.java

```diff
@@ -10,16 +13,14 @@ public class OidcClientReactiveFilterConfigFallbackInterceptor extends FallbackC
 
     private static final String OLD_PREFIX = "quarkus.oidc-client-reactive-filter.";
     private static final String NEW_PREFIX = "quarkus.rest-client-oidc-filter.";
-
-    public OidcClientReactiveFilterConfigFallbackInterceptor() {
-        super(OidcClientReactiveFilterConfigFallbackInterceptor::rename);
-    }
-
-    private static String rename(String originalName) {
-        if (!originalName.startsWith(NEW_PREFIX)) {
-            return originalName;
+    private static final Function<String, String> RENAME_FUNCTION = new Function<String, String>() {
+        @Override
+        public String apply(String s) {
+            return StringUtil.changePrefix(s, NEW_PREFIX, OLD_PREFIX);
```

Comment:
@gsmet My ide has been smarter than me and I fix it back (making it wrong) later on, uff :/ many thanks to have found it!